parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -467,6 +467,79 @@ renderTypeError e env src curPath = case e of
         annotatedAsErrorSite src abilityCheckFailureSite,
         debugSummary note
       ]
+  AbilityEqFailure {..} ->
+    mconcat
+      [ "I could not solve an ability equation when checking the expression ",
```

Comment:
Is there a user-friendlier description we could use here? Or do we like this terminology?

Reply:
Yeah, I'd just do: "I found an abilities mismatch here:"